crates/lib/kajiya/src/world_renderer.rs

```diff
@@ -671,10 +667,9 @@ impl WorldRenderer {
         }
     }
 
-    pub fn set_instance_transform(&mut self, inst: InstanceHandle, position: Vec3, rotation: Quat) {
+    pub fn set_instance_transform(&mut self, inst: InstanceHandle, transform: Affine3A) {
```

Comment:
In addition to this, `fn add_instance should also be taking an `Affine3A` instead of the separate position and rotation.